tests/test_integration.py

```diff
@@ -74,12 +69,23 @@ class Meta:
 
 class TestSwagger(unittest.TestCase):
 
-    def test_swagger_valid(self):
+    def test_swagger_without_cache(self):
+        # Test if Swagger API documentation loads
+        # when cache all requests is disabled
+        os.environ['CACHE_ALL_REQUESTS'] is False
```

Comment:
As noted in a previous comment, we need to keep the value as a string.  We also want to set the environment variable here, not perform an equality check, so the statement should read as follows:

```python
os.environ['CACHE_ALL_REQUESTS'] = 'False'
```

Reply:
I think this is what i had before. I am happy to revert the chagnes :)

https://github.com/fecgov/openFEC/pull/3140/files/d189ac0bb68faef0b253d36cf5df73bf96e347c6#diff-7eb47fc18d4a6196f0940784484a5bf3R78

Reply:
First i have to set the env variable and then  evaluate
 os.environ['CACHE_ALL_REQUESTS'] = 'False'?

Reply:
It is what you had before, and this statement is also what is setting the variable; that's all that needs to happen in the test as the code being tested handles the rest. :-)

Though, in thinking about this a bit more, this is where this may need to shift a bit; you and @lbeaufort both pointed out that this is just testing the validation of the generated API documentation itself, not actually going to the documentation URL and loading it up.

I'm sorry for the confusion, it's in part just due to lack of time to be able to focus.  In any case, these tests ought to be re-written to load the Swagger documentation itself by attempting to visit the documentation URL and seeing if a `200` response is given or something to that effect.  Given these proposed changes to the tests, we should keep the one test as we had it originally to validate the Swagger UI itself.

@lbeaufort does this make sense to you, based on what we discussed?

Reply:
@ccostino - that is what I was thinking too. I'm sorry I haven't spoken up sooner, I've been swamped.